x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/policy_tabs.tsx

```diff
@@ -74,8 +77,26 @@ export const PolicyTabs = React.memo(() => {
           </>
         ),
       },
+      {
+        id: 'hostIsolationExceptions',
+        name: i18n.translate(
+          'xpack.securitySolution.endpoint.policy.details.tabs.isInHostIsolationExceptions',
+          {
+            defaultMessage: 'Host isolation exceptions',
+          }
+        ),
+        content: (
+          <>
+            <EuiSpacer />
+            <PolicyHostIsolationExceptionsTab
+              policyId={policyId}
```

Comment:
Why do you need the policyId? It is inside the policyItem right?

Reply:
indeed. I wrote this part when the `policy` object was an optional prop. I'll change it.